library/Cube/Ido/IdoHostStatusCube.php

```diff
@@ -17,8 +17,12 @@ public function getAvailableFactColumns()
     {
         return array(
             'hosts_cnt'           => 'COUNT(*)',
-            'hosts_nok'           => 'SUM(CASE WHEN hs.current_state = 0 THEN 0 ELSE 1 END)',
-            'hosts_unhandled_nok' => 'SUM(CASE WHEN hs.current_state != 0'
+            'hosts_nok'           => 'SUM(CASE WHEN hs.current_state = 1 THEN 1 ELSE 0 END)',
+            'hosts_unhandled_nok' => 'SUM(CASE WHEN hs.current_state = 1'
```

Comment:
Please get rid of `nok` here as well. Use `hosts_down` and `hosts_down_unhandled` instead.